Updates to New Relic Docs

dansjackson opened the pull request:
I've updated the new relic docs to allow the user to specify the location of the PHP versions if multiple PHP versions are installed or if the PHP version is not in the correct location

dansjackson committed:
Merge pull request #1 from ukfast/master

updates

dansjackson committed:
Merge pull request #2 from ukfast/master

Pull to repo

dansjackson committed:
added guide on how to define PHP install path

terencesimms commented on source/monitoring/installnewrelic.md:
````diff
@@ -34,6 +34,16 @@ New Relic offers a [free tier account](https://newrelic.com/signup/) with "100 G
  $ newrelic-install install
  ```
 
+```eval_rst
````
I would change this to not be a note, and instead be under a new subheading such as # Installing the agent on Plesk/cPanel 
Mention that Plesk and cPanel install PHP binaries to alternate locations, so you will need to use the following commands to ensure these are picked up. This will also apply to servers with non-default PHP installation paths.

Then, provide the example for Plesk also. If you check in the UKFast testing account, one of the servers is a Plesk test server,

terencesimms reviewed (changes requested):
I think the title of this article should be changed to 
'How to Install the New Relic Agent on CentOS', in line with other guides.

dansjackson committed:
changed the layout of the information

dansjackson committed:
made the KB more accessible

dansjackson committed:
more formatting fixes

terencesimms committed:
Update installnewrelic.md

amending service commands, adding panel note about restarting services, removing nginx restart (no need) & tweaking a sentence, plus example subheadings for linkability
